docs/release-notes/NuGet-6.4.md

```diff
@@ -0,0 +1,109 @@
+---
+title: NuGet 6.4 Release Notes
+description: Release notes for NuGet 6.4 including new features, bug fixes, and DCRs.
+author: jebriede
+ms.author: jebriede
+ms.date: 10/27/2022
+ms.topic: conceptual
+---
+
+# NuGet 6.4 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**6.4**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.4](https://visualstudio.microsoft.com/downloads/) | [7.0.100](https://dotnet.microsoft.com/download/dotnet-core/<SDKMajorMinorVersionOnly>)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 17.4 with .NET Core workload
+
+## Summary: What's New in 6.4
+
+* [DCR]: Package Source Mapping API does not support saving  - [#11935](https://github.com/NuGet/Home/issues/11935)
```

Comment:
Given that we're not shipping the source mapping PM UI feature until 17.5 we should move this to the long fixes list. 
It doesn't need a special call out just yet. :)